var/spack/repos/builtin/packages/gromacs/package.py

```diff
@@ -25,6 +25,7 @@ class Gromacs(CMakePackage):
     maintainers = ['junghans', 'marvinbernhardt']
 
     version('master', branch='master')
+
```

Comment:
Was this change intentional or accidental?

Reply:
Those changes are accidental.

Reply:
Can you remove these changes from the PR?